src/parsers/motoko-tt-parse/preprocess.ts

```diff
@@ -78,11 +78,21 @@ export default function preprocess(
                     indent++;
                 }
 
+                // Find next non-empty line index
```

Comment:
do comment-only lines count as empty?

Reply:
Oh the test says "yes"!

Reply:
Oh, yes I'll clarify that. Empty after filtering out comments.